routing/dht/dht.go

```diff
@@ -126,15 +125,7 @@ func (dht *IpfsDHT) putValueToPeer(ctx context.Context, p peer.ID,
 
 	pmes := pb.NewMessage(pb.Message_PUT_VALUE, string(key), 0)
 	pmes.Record = rec
-	rpmes, err := dht.sendRequest(ctx, p, pmes)
-	if err != nil {
-		return err
-	}
-
-	if !bytes.Equal(rpmes.GetRecord().Value, pmes.GetRecord().Value) {
-		return errors.New("value not put correctly")
-	}
-	return nil
```

Comment:
does this mean the dht is currently erroring a lot? (i.e. cause we stopped responding in the other put value?) 

from a protocol standpoint, we do lose the ability to sanity check whether things are going wrong without this response. :/ -- its fine, but i think it may not help us find bugs.